sdk/src/main/scala/io/horizen/account/api/http/route/AccountEthRpcRejectionHandler.scala

```diff
@@ -0,0 +1,26 @@
+package io.horizen.account.api.http.route
+
+import akka.http.scaladsl.server.{MalformedRequestContentRejection, RejectionHandler}
+import io.horizen.account.api.rpc.request.RpcId
+import io.horizen.account.api.rpc.response.RpcResponseError
+import io.horizen.account.api.rpc.utils.{RpcCode, RpcError}
+import io.horizen.account.serialization.EthJsonMapper
+import io.horizen.api.http.SidechainApiResponse
+
+object AccountEthRpcRejectionHandler {
+
+  def rejectionHandler: RejectionHandler =
+    RejectionHandler.newBuilder()
+      .handle {
+        case MalformedRequestContentRejection(msg, _) =>
+          SidechainApiResponse(
+            EthJsonMapper.serialize(
+              new RpcResponseError(
+                new RpcId(),
+                RpcError.fromCode(RpcCode.ParseError, msg)
```

Comment:
here we return error code `ParseError(-32700, "Parse error")`
infura example from Cronic returns `{"jsonrpc":"2.0","error":{"code":-32600,"message":"invalid json request"}}` which in our case is `InvalidRequest(-32600, "Invalid request")`

Let's double check what is actually expected.

Reply:
Decided to go with -32700, as it's description fully represent this case. Notified Cronicc about it.
![image](https://user-images.githubusercontent.com/64777406/232028508-24443744-5719-4d13-9103-8376ab954f5f.png)